drivers/Makefile.dep

```diff
@@ -32,6 +32,21 @@ ifneq (,$(filter cc110x,$(USEMODULE)))
   endif
 endif
 
+ifneq (,$(filter cc2420,$(USEMODULE)))
+  USEMODULE += cc2420
```

Comment:
Oh yeah, it was also used with `at86rf2xx` so I added it, but you're right, it's not required. Should I remove it?